src/index.template.html

```diff
@@ -249,9 +192,16 @@
             window.location.reload();
           }
         }
-      //If in 5 seconds it hasn't render the root of the page thne reload the page.
+      //If in 5 seconds it hasn't render the root of the page then reload the page.
       setTimeout(forceRefreshIfNotloaded, 30000);
+
+      window.addEventListener('load', function (event) {
+        if (!('noModule' in HTMLScriptElement.prototype)) {
+          document.getElementById('error').innerHTML =
+            "<b>⚠️ Error!</b> The version of this browser doesn't support some essential features used in InterPro.<br/><br/>Please upgrade your browser.";
+          console.error('No support for es-modules');
```

Comment:
Is it necessary for end-users?

Reply:
no necessary, but thought of leaving it there, for debugging RT tickets.